tests/test_windows.py

```diff
@@ -292,3 +292,17 @@ def test_intersection():
     """Window intersection works."""
     window = intersection(Window(0, 0, 10, 10), Window(8, 8, 12, 12))
     assert window == ((8, 10), (8, 10))
+
+
+def test_round_window_to_full_tiles():
+    with rasterio.open('tests/data/alpha.tif') as src:
+        block_shapes = src.block_shapes
+        test_window = ((321, 548), (432, 765))
```

Comment:
Ah, it's dependent on the division behavior of the python version

Python 3.5
```
In [4]: tuple(round_window_to_full_tiles(((-1, 391), (-1,405)), ((6, 400), )))
Out[4]: ((0, 396), (0, 800))
```

Python 2.7
```
In [4]: tuple(round_window_to_full_tiles(((-1, 391), (-1,405)), ((6, 400), )))
Out[4]: ((-6, 396), (-400, 800))
```

So we'll need to add a `from __future__ import division` at the top and adjust the code accordingly. There are no other division operators in this file so nothing else should be affected.